test/config.ini.in

```diff
@@ -23,3 +23,4 @@ RPCAUTH=@abs_top_srcdir@/share/rpcauth/rpcauth.py
 @BUILD_BITCOIND_TRUE@ENABLE_BITCOIND=true
 @ENABLE_FUZZ_TRUE@ENABLE_FUZZ=true
 @ENABLE_ZMQ_TRUE@ENABLE_ZMQ=true
+@ENABLE_EXTERNAL_SIGNER_TRUE@ENABLE_EXTERNAL_SIGNER=true
```

Comment:
It seems the `@ENABLE_EXTERNAL_SIGNER_TRUE@` substitution does not work for some reasons.

Reply:
nm, forgot to run `./autogen.sh`